src/app/components/sectioned-form/index.html

```diff
@@ -11,16 +11,30 @@
   </sky-demo-page-summary>
 
   <sky-demo-page-properties sectionHeading="Sectioned form section settings">
+    <sky-demo-page-property
+      propertyName="heading"
+    >
+    Specifies the section header.
```

Comment:
In the description for this property, add "This property accepts `string` values."